api/utilities/model_serializers/attribute.py

```
@@ -54,3 +56,33 @@ def create_attribute(self, data):
         data['_key'] = case(data['_key'])
 
         return Attribute(**data)
+
+
+class UpdateAttributeSchema(BaseSchema):
+    """Update Attribute model schema"""
+
+    id = fields.String(
+        missing='None',
```

why is this set to `None` by default?

---

@brayoh This could be anything, but I chose to use `'None'` because I want to return this error message `attribute with the id of None is not related to the asset category of id {asset_category_id}` if no id was provided.